content/en/docs/prologue/quick-start.md

```diff
@@ -119,7 +129,7 @@ sudo setcap cap_net_admin+ep ./cloud-hypervisor/target/release/cloud-hypervisor
 	--cpus boot=4 \
 	--memory size=1024M \
 	--net "tap=,mac=,ip=,mask=" \
-	--rng
+	--rng src=/dev/urandom
```

Comment:
I think you can just remove this option. It's on by default and users almost never need to change it.

Reply:
Oh that would make sense. I only had it because I was following these docs

Reply:
Ok, it should be ready for more review if you're happy with it.